package.json

```diff
@@ -35,19 +36,22 @@
     "ember-cli-mocha": "0.10.1",
     "ember-cli-notifications": "3.2.0",
     "ember-cli-uglify": "1.2.0",
+    "ember-computed-decorators": "0.2.2",
     "ember-data": "^2.4.0",
     "ember-disable-proxy-controllers": "1.0.1",
     "ember-export-application-global": "1.0.5",
-    "ember-frost-core": ">=0.0.14 <2.0.0",
+    "ember-frost-core": "0.17.3",
```

Comment:
should update core to at least 0.20.0 so you can  implement the ember-hook stuff here in the future.